instrumentation/opentelemetry-instrumentation-logging/src/opentelemetry/instrumentation/logging/__init__.py

```diff
@@ -144,7 +144,12 @@ def record_factory(*args, **kwargs):
             )
             log_level = log_level or logging.INFO
 
-            logging.basicConfig(format=log_format, level=log_level)
+            root_logger = logging.getLogger()
+            if root_logger.hasHandlers():
+                for handler in root_logger.handlers:
+                    handler.setFormatter(logging.Formatter(log_format))
```

Comment:
What about `log_level`?

Reply:
@ocelotl Is there a reason to review this pr and not the more recent one [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2384)?

Reply:
@ocelotl, as @dhofstetter, said lets go with their newer and more complete pr. i meant this as more a discussion starter. 

thanks for the review though. 

also ill close this pr so its not confusing anymore